.pre-commit-config.yaml

```diff
@@ -1,79 +1,79 @@
+# "exclude: '^$'" specifies the global file exclude pattern; '^$' excludes
 
 fail_fast: false
+# "fail_fast: false" means pre-commit continues to run hooks after the first failure
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
-    hooks:
-    - id: no-commit-to-branch
-      args: [--branch, master, --branch, develop]
-    - id: trailing-whitespace
-    - id: check-added-large-files
-      args: [maxkb=512]
-    - id: check-ast
-    - id: check-json
-    - id: pretty-format-json
-      args: [--autofix]
-    - id: check-xml
-    - id: check-yaml
-    - id: check-symlinks
-    - id: check-merge-conflict
-    - id: detect-private-key
-    - id: check-executables-have-shebangs
-    - id: name-tests-test
-      # args: ['--django']        # uncomment to change tests to 'test_*.py' rather '*_test.py'
-    - id: requirements-txt-fixer  # Sorts the requirement.txt files' content
-      files: 'requirements/requirements.*\.txt$'
-    - id: trailing-whitespace
+    -   repo: https://github.com/pre-commit/mirrors-mypy
+        rev: v0.971
+        hooks:
+            -   id: mypy  # Run mypy type checker; note: runs with flag --missing-imports
+                exclude: ondewo/
+                args: [ --config-file=mypy.ini ]
+                # Use language: system to run Mypy git repo's existing virtual environment
+                # instead of mypy's own virtual environment. Necessary to find types-.* packages
+                # Replace language: python in your hook definition with language: system.
+                # Remove the additional_dependencies line and install Mypy into your
+                # environment directly. Now, Pre-commit will not create a separate virtualenv
+                # for the hook and will run it in whatever environment you happen to be in when
+                # you run git commit or pre-commit run => https://jaredkhan.com/blog/mypy-pre-commit
+                language: system
+                # use require_serial so that script
+                # is only called once per commit
+                require_serial: true
+                # Print the number of files as sanity-check
+                verbose: true
+                types: [ python ]
 
-  ## consider using seed-isort-config
-  ## (pip install seed-isort-config)
-  ##
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: 'v5.8.0'
-    hooks:
-    - id: isort
-      args: ["--profile", "black"]
+    -   repo: https://github.com/pre-commit/mirrors-autopep8
+        rev: v1.6.0
+        hooks:
+            -   id: autopep8  # Perform pep8 auto-formatting
+                args: [ --max-line-length=120, --in-place ]
 
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: 'v2.2.0'
-    hooks:
-    - id: seed-isort-config
+    -   repo: https://github.com/pre-commit/pre-commit-hooks
+        rev: v4.3.0
+        hooks:
+            -   id: check-added-large-files  # Prevent giant files from being committed
```

Comment:
What happened to `    - id: no-commit-to-branch
      args: [--branch, master, --branch, develop]`?